src/GraphViewer.jsx

```diff
@@ -55,12 +55,12 @@ export default class GraphViewer extends React.Component {
         }
 
         const displayLabels = isInstructor ||
-              this.props.gAssignmentType === 0 ||
-              this.props.gAssignmentType === 1;
+            this.props.gAssignmentType === 0 ||
+            this.props.gAssignmentType === 1;
```

Comment:
Through debugging the problem I'm seeing, I'm realizing there's another cleanup we can do here: there's a bunch of setup code like this happening in the component's `render()` method. That's fine, but anything that can be handled in the constructor should be handled there, instead of render. Anything that doesn't rely on DOM elements should be moved to the constructor here. That can be handled as a separate PR, I'm just making a note of it here.